frontend/__tests__/components/chat/chat-interface.test.tsx

```diff
@@ -281,6 +281,31 @@ describe.skip("ChatInterface", () => {
     expect(within(error).getByText("Something went wrong")).toBeInTheDocument();
   });
 
+  it("should render both GitHub buttons when ghToken is available", () => {
+    vi.mock("@remix-run/react", async (importActual) => ({
+      ...(await importActual<typeof import("@remix-run/react")>()),
+      useRouteLoaderData: vi.fn(() => ({ ghToken: "test-token" })),
+    }));
+
+    const messages: Message[] = [
+      {
+        sender: "assistant",
+        content: "Hello",
+        imageUrls: [],
+        timestamp: new Date().toISOString(),
+      },
+    ];
+    renderChatInterface(messages);
+
+    const pushButton = screen.getByRole("button", { name: "Push to GitHub" });
+    const prButton = screen.getByRole("button", { name: "Push & Create PR" });
```

Comment:
I think once the `Push & Create PR` button has been clicked once, we should only show one button after that. The agent (and user) will get v confused if you push and create PR, and then click "push to branch", since the PR is already opened.

Once you click "Push & Create PR" we should just have one button that says "Push changes to PR"

Reply:
done the most recent commit